jupyterhub/auth.py

```diff
@@ -298,27 +298,28 @@ def validate_username(self, username):
 
         This function is called after the user has passed all authentication checks
         and is ready to successfully authenticate. This function must return the
-        authentication dict reguardless of changes to it.
+        auth_model dict reguardless of changes to it.
```

Comment:
By changing the name, we are making it explicit that we will pass these args _positionally_ (because if we passed it by keyword, this would be a breaking change). Maybe add that fact to the help string?